sourced/ml/transformers/basic.py

```diff
@@ -300,16 +300,16 @@ def __call__(self, rows: RDD) -> RDD:
         return rows.flatMap(self.deserialize_uast)
 
     def deserialize_uast(self, row: Row):
-        if not row[EngineConstants.Columns.Uast]:
+        if EngineConstants.Columns.Uast not in row:
             return
```

Comment:
I think we should keep both conditions because `row[EngineConstants.Columns.Uast]` can be an empty list and we do not want an empty list further:
```python
if EngineConstants.Columns.Uast not in row or not row[EngineConstants.Columns.Uast]:
    return
```

Reply:
I did this because the code returned a ValueError when the dataframe did not contain the "uast" column https://github.com/src-d/ml/pull/301#discussion_r212960068
Do you want this : `if EngineConstants.Columns.Uast not in row and not row[EngineConstants.Columns.Uast]` ?

Reply:
ok, let's put both conditions, I am ok with that.